frontend/src/components/CharacterPage.jsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import Navbar from './Navbar';
+import { Line } from 'react-chartjs-2';
+import 'chart.js/auto';
+
+const CharacterPage = () => {
+    const { character } = useParams();
+    const [stats, setStats] = useState(null);
+
+    useEffect(() => {
+        const fetchStats = async () => {
+            try {
+                const response = await fetch(`http://localhost:3000/stats/${character}`);
+                const data = await response.json();
+                setStats(data);
+            } catch (error) {
+                console.error('Error fetching character stats:', error);
+            }
+        };
+
+        fetchStats();
+    }, [character]);
+
+    const pickrateGraphData = {
+        labels: stats?.pickrateHistory.map(data => new Date(data.timestamp).toLocaleDateString()) || [],
+        datasets: [
+            {
+                label: 'Pickrate',
+                data: stats?.pickrateHistory.map(data => data.value) || [],
+                borderColor: 'rgba(75, 192, 192, 1)',
+                backgroundColor: 'rgba(75, 192, 192, 0.2)',
```

Comment:
generally it's good to store rgb values like this in a variable and name it something descriptive, like
` redColor = rgba(...)` so people reading the code know what the value looks like